nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -128,8 +132,7 @@ class ConfigEncryptionTool {
     private static final String NEW_FLOW_PROVIDER_ARG = "newFlowProvider"
     private static final String TRANSLATE_CLI_ARG = "translateCli"
 
-    private static final String PROTECTION_SCHEME_DESC = String.format("Selects the protection scheme for encrypted properties.  " +
-            "Valid values are: [%s] (default is %s)", PropertyProtectionScheme.values().join(", "), DEFAULT_PROTECTION_SCHEME.name())
+    private static final String PROTECTION_SCHEME_DESC = "Selects the protection scheme for encrypted properties. Default is AES_GCM"
```

Comment:
That's a good question.  On the one hand, based on the configured properties, many of the possible protection schemes might not be available.  A more complex solution could attempt to drive the output based on the supported providers information when the tool executes.  Maintaining the current solution could be done through an extension of the ProtectionSchemeResolver.  The valid values are described in Toolkit Guide documentation, but of course that is not as convenient as writing the values in the output.  Do you think keeping something similar to the current approach is the best solution?

Reply:
Yes -- I see how it could be a bit misleading to list schemes that are not technically enabled, but I think the resulting error would at least give feedback that would allow the user to configure it if needed.  I think the benefit of having the full list in the CLI help description is worth it.

Reply:
Sounds good, I will make changes here and in the other CLI references to write the valid values.